packages/analytics/amplify_analytics_pinpoint/example/.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,134 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
I don't think we should be checking in anything in the `.idea` directory.

Reply:
Thanks I have deleted that directory